include/circt-c/RTLDialect.h

```diff
@@ -0,0 +1,30 @@
+/*===-- circt-c/RTLDialect.h - C API for RTL dialect --------------*- C -*-===*\
+|*                                                                            *|
+*|
+\*===----------------------------------------------------------------------===*/
+
+#ifndef CIRCT_C_RTLDIALECT_H
+#define CIRCT_C_RTLDIALECT_H
+
+#include "mlir-c/IR.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Registers the RTL dialect with the given context. This allows the
```

Comment:
Ok, I see the MLIR C bindings are doing the same thing, but this still seems a bit weird to me:  The use of /**/ comments is very inclusive to old C compilers, but I think that BCPL style // comments were part of ansi c 89/90, so I think they are ok to use.  Doing so gets us better alignment with the c++ side of things.  Could you ask the MLIR forum what the rationale here is, and whether using // would be ok with them?

Reply:
https://llvm.discourse.group/t/a-quick-question-about-comment-styles/2144

Seems like there is a policy, and I'm not planning on rocking the boat too much yet :)